src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -184,7 +184,13 @@ public void editFeedbackResponseComment(String commentIdSuffix, String newCommen
     }
     
     public boolean verifyAllResultsPanelBodyVisibility(boolean visible){
-        for(WebElement e : browser.driver.findElements(By.cssSelector(".panel-heading+.panel-body"))){
+        int numOfQns = browser.driver.findElements(By.cssSelector(".panel-heading+.panel-collapse")).size();
+        assertTrue(numOfQns > 0);
+        
+        // Wait for the total duration according to the number of collapse/expand intervals between questions
+        ThreadHelper.waitFor(numOfQns * 50);
```

Comment:
i think we might need to add a slightly longer wait. Using the exact timing may still cause failures on slower computers. Maybe +1 second to it?